src/main/java/bio/terra/service/filedata/flight/delete/DeleteFileAzureDirectoryStep.java

```diff
@@ -27,7 +29,9 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
     AzureStorageAuthInfo storageAuthInfo =
         workingMap.get(FileMapKeys.STORAGE_AUTH_INFO, AzureStorageAuthInfo.class);
     try {
-      boolean found = tableDao.deleteDirectoryEntry(fileId, storageAuthInfo);
+      boolean found =
+          tableDao.deleteDirectoryEntry(
+              fileId, storageAuthInfo, dataset.getId(), DATASET_TABLE.toTableName());
```

Comment:
A future refactor: since the last argument to `TableDao.deleteDirectoryEntry()` is always `StorageTableName.toTableName()`, you can make `deleteDirectoryEntry()` accept `StorageTableName` instead of `String` and call `toTableName()` in the method. There may be other cases like this in the code.

Reply:
Yeah, it's a little wonky right now b/c for a lot of the methods I need to be able to handle both .toTableName() and .toTableName(resourceId). So, passing around the actual table name seemed the most straight forward solution for now. In the future, when they all accept a resource id, I can move this code closer to where the table name is used.

Reply:
Sounds good!